tests/std/tests/P0088R3_variant/test.cpp

```diff
@@ -6617,10 +6617,10 @@ namespace msvc {
         //
         //===----------------------------------------------------------------------===//
 
-        enum CallType : unsigned { CT_None, CT_NonConst = 1, CT_Const = 2, CT_LValue = 4, CT_RValue = 8 };
+        enum CallType : unsigned int { CT_None, CT_NonConst = 1, CT_Const = 2, CT_LValue = 4, CT_RValue = 8 };

```

Comment:
Ditto "this test is mostly mechanically extracted", but not all of these changes need to be reverted. Everything after line 6365 in this file is local, so this change and the next can stay, but the changes to earlier lines should be reverted.

Reply:
Thank you for explaining

Reply:
I apparently did a poor job of explaining. This test has several thousand lines that are mechanically extracted from libc++, and a few hundred lines at the end that we've added on. _This_ change was in the added on bit - and therefore good - but all the changes earlier in the file were in the extraction.

I've pushed a commit to restore just this change.